druid/src/event.rs

```diff
@@ -364,18 +370,25 @@ pub enum InternalLifeCycle {
     DebugInspectState(StateCheckFn),
 }
 
-/// Information about the widgets surroundings.
+/// Information about the widget's surroundings.
+///
+/// The global origin is also saved in the widget state.
+///
+/// When the ViewContext of widget changes it receives a `ViewContextChanged` event.
```

Comment:
Let's put `ViewContext` into quotes and add an indefinite article before *widget* - *When the `ViewContext` of **a** widget .....*